src/sysusers/sysusers.c

```diff
@@ -1814,9 +1821,12 @@ static int parse_line(const char *fname, unsigned line, const char *buffer) {
                         return r;
                 if (r == 0)
                         log_syntax(NULL, LOG_WARNING, fname, line, 0,
-                                   "Conflict with earlier configuration for %s '%s', ignoring line.",
-                                   item_type_to_string(i->type), i->name);
-
+                                   "Conflict with earlier configuration for %s '%s' %s%s%s%.0u, ignoring line.",
+                                   item_type_to_string(i->type), i->name,
+                                   existing->filename ? "in " : "",
+                                   existing->filename ?: "",
+                                   existing->filename ? ":" : "",
+                                   existing->filename ? existing->line : 0);
```

Comment:
honestly, I am not a big fan of the .0 stuff, and so many conditionalizations. I think it would be much prettier to just do two two distinct log_syntax() calls, one with the existing info, one without